manifests/e/ElectronicArts/EAapp/12.218.0.5475/ElectronicArts.EAapp.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ElectronicArts.EAapp

```

```suggestion
PackageIdentifier: ElectronicArts.EADesktop
```
This package already exists under the EADesktop name, this new version should be added there